euphoria-core/src/main/java/cz/seznam/euphoria/core/client/dataset/windowing/GlobalWindowing.java

```diff
@@ -50,27 +53,27 @@ private Object readResolve() throws ObjectStreamException {
     }
 
     @Override
-    public int compareTo(BatchWindow o) {
+    public int compareTo(Window o) {
```

Comment:
hm, i think it is. `Window` here refers here to `GlobalWindowing.Window`; i'll make that explicit.